pkg/client/cli/cmd_status.go

```diff
@@ -5,17 +5,19 @@ import (
 	"io"
 
 	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
```

Comment:
Don't think you need version parsing (see comment further down), and if you do, you should use `github.com/blang/semver` because that's what the rest of the code base is using.